src/base/Worker.ts

```diff
@@ -554,7 +554,7 @@ export default class Worker<
   /**
    * Start the job execution
    */
-  public start = async () => {
+  public start = () => {
```

Comment:
Minor: so technically in the core where the workers are started, it should no longer be awaited, right? (doesn't affect functionality though)

Reply:
Thanks, thanks, yes, it's probably removed in one of my stashes with the periodic syncer logic.